db_handle/manage_portfolio.py

```diff
@@ -0,0 +1,112 @@
+import sys
+sys.path.append(r'.')
+from db_handle import postgres_conn
+import random, string
+
+
+"""Init connection to the database"""
+postgres_conn.admin_client()
+
+def create_new_category():
+    """Create ranomly generated category ID"""
+    category_id = []
+    for i in range(1, 7):
+        category_id.append(random.choice(string.digits))
+    category_id = 'CAT-' + ''.join(category_id)
+
+    """Enter the category name"""
+    category_name = input("Enter category name: ")
+
+    # """Calculates the total sub-categories / for new categories is 0 by default"""
```

Comment:
@rayapetkova We won't use this section with the comments. Could you remove this block of code from the function.